src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
Do you need these styles for the card? Block elements displayed vertically be default